### PR TITLE
Add documentation for configuring a HTTP proxy for requests

### PR DESCRIPTION
**Pull request by bastilian**
Documentation for the new HTTP proxy settings: https://github.com/theforeman/foreman/pull/4640

### CODE REVIEW

File: _includes/manuals/1.16-develop/5.9_http_proxy.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+If Foreman is running behind a Firewall, that has HTTP port 80 and HTTPS port 443 blocked a HTTP proxy can be configured to allow requests to external compute resources, such as AWS EC2, Rackspace or Docker as well as external content provider and registries.
```
Comment (ares) [inline _includes/manuals/1.16-develop/5.9_http_proxy.md]: It does not have to be port 80/443, it applies to all communication with external systems, including foreman proxies. I don't dare to suggest the sentence, but I think it should mention all external HTTP communication can be proxied and include few examples such as compute resource, foreman proxy, other endpoints coming from plugins e.g. docker registry.

**Comment by mmoll** [comment]
@bastilian also, please rebase :)

**Comment by ares** [comment]
@bastilian ping

**Comment by bastilian** [comment]
@mmoll rebased.
@ares I changed the first sentence marginally. Any suggestion to make the documentation more readable and understandable is welcome! :)

**Comment by ares** [comment]
Hm, the 1.16-develop does not seem to be included and we don't have 1.17 copy yet, I'm afraid we'll have to wait until we have 1.17 created. When I tried to generate pages, it failed to load the page since 1.16-develop is not searched. Otherwise good to go.

**Review by dLobatog — CHANGES_REQUESTED** [review]
@bastilian Please rebase so that 5.9_http_proxy is in manuals/1.16 instead of 1.16-develop, should be good to go after that. Thanks !

**Comment by bastilian** [comment]
@dLobatog moved the docs into the 1.16 manuals.

**Review by dLobatog — APPROVED** [review]
Thanks @bastilian !

**Pull request merged**